### PR TITLE
Refine oscar debugging

### PR DESCRIPTION
**Pull request by chaokunyang**
## What do these changes do?
Refine oscar debugging so that we can locate hang issues more easily.

### CODE REVIEW

File: mars/oscar/core.pyx [hunk 1]
```diff
@@ -263,17 +263,17 @@ cdef class _BaseActor:
...
         try:
             res = None
             while True:
-                async with self._lock:
-                    if not is_exception:
-                        res = await gen.asend(res)
-                    else:
-                        res = await gen.athrow(*res)
+                with debug_async_timeout('lock_hold_timeout',
+                                         'async_generator % hold lock timeout', gen):
```
Comment (wjsi) [inline mars/oscar/core.pyx:271]: `%r`?

File: mars/oscar/debug.py [hunk 2]
```diff
@@ -30,6 +31,8 @@
 @dataclass
 class DebugOptions:
     actor_call_timeout: int = 10
+    process_message_timeout: int = 30
+    lock_hold_timeout: int = 30
```
Comment (wjsi) [inline mars/oscar/debug.py:35]: Rename to `actor_lock_timeout`?

**Review by wjsi — DISMISSED** [review]
LGTM

**Review by wjsi — APPROVED** [review]
LGTM

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**